src/components/fullHeight/index.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import { Box } from '../box';
+
+import { FullHeight } from './index';
+
+const meta: Meta<typeof FullHeight> = {
+  title: 'Layout/Full Height',
+  component: FullHeight,
+
+  parameters: {
+    layout: 'fullscreen',
+  },
+
+  args: {
+    children: (
+      <Box backgroundColor="brand.primary" minHeight="screen-height">
+        I am full height container
+      </Box>
```

Comment:
This needs to be passed using `mapping` to avoid issues with storybook. As documented [here](https://github.com/smg-automotive/components-pkg/pull/1206/commits/4e3e81d3f9f7f17e24b85110a3edaf5d15e54ec2#r1844108779)